apps/web/components/examples/blockchain/advanced-learning-platform.tsx

```
@@ -0,0 +1,756 @@
+      )}
+
+      {/* Real-time Events Feed */}
+      {enableRealTimeSync && realtimeEvents.length > 0 && (
+        <div className="bg-white rounded-lg border border-gray-200 p-6">
+          <h3 className="font-medium text-gray-900 mb-4 flex items-center gap-2">
+            <BoltIcon className="h-5 w-5" />
+            Recent Activity
+          </h3>
+          <div className="space-y-2 max-h-40 overflow-y-auto">
+            {realtimeEvents.slice(-5).reverse().map((event, index) => (
+              <div key={index} className="text-sm text-gray-600 flex items-center gap-2">
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)